mirage/config.ts

```diff
@@ -225,6 +229,10 @@ export default function(this: Server) {
 
     this.get('/users/:id/nodes', userNodeList);
     this.get('/sparse/users/:id/nodes', userNodeList);
+    this.get('/users/:id/registrations', userRegistrationList);
+    osfNestedResource(this, 'user', 'draftRegistrations', {
```

Comment:
So there were a couple of ways to go on these. I made the userRegistrationList like the userNodeList because it's most correct, as it's searching for registrations that have the user as a contributor, and we have plenty of those set up in mirage. For the draftRegistrations, we're not yet managing contributors on the drafts, so it didn't make sense to do it that way yet, so I just went with the standard `osfNestedResource` instead.